File: 100-Days-Series/Day 81/EvenOddSum.py

# Function to calculate the sum of even integers.
def get_sum_of_even_numbers(numbers):
    evenSum=0
    for i in range(0,len(numbers)):
        if numbers[i]%2==0:
            evenSum+=numbers[i]
    return evenSum

# Function to calculate the sum of odd integers.
def get_sum_of_odd_numbers(numbers):
    oddSum=0
    for i in range(0,len(numbers)):
        if numbers[i]%2!=0:
            oddSum+=numbers[i]
    return oddSum

# Let's Start Here
n=int(input('Enter the number of integers : '))
numbers=[]
for i in range(0,n):
    print('Enter',(i+1),' integer : ',end="")
    num=int(input())
    numbers.append(num)

evenSum=get_sum_of_even_numbers(numbers)
oddSum=get_sum_of_odd_numbers(numbers)
print('The sum of even integers is : ',evenSum)
print('The sum of odd integers is : ',oddSum)

File: 100-Days-Series/Day 81/EvenOddSum.java

public class EvenOddSum {
    // Method to calculate the sum of even numbers.
    public static int getSumOfEvenNumber(int numbers[]){
        int evenSum=0;
        for(int i=0;i<numbers.length;i++){
            if(numbers[i]%2==0){
                evenSum+=numbers[i];
            }
        }
        return evenSum;
    }
    // Method to calculate the sum of odd numbers.
    public static int getSumOfOddNumber(int numbers[]){
        int oddSum=0;
        for(int i=0;i<numbers.length;i++){
            if(numbers[i]%2!=0){
                oddSum+=numbers[i];
            }
        }
        return oddSum;
    }
    // Main Method 
    public static void main(String [] args){
        int n;
        java.util.Scanner sc=new java.util.Scanner(System.in);
        System.out.print("Enter the number of integers : ");
        n=sc.nextInt();
        int []numbers=new int[n];
        for(int i=0;i<n;i++){
            System.out.print("Enter "+(i+1)+" integer : ");
            numbers[i]=sc.nextInt();
        }
        int evenSum=getSumOfEvenNumber(numbers);
        int oddSum=getSumOfOddNumber(numbers);
        System.out.println("The Sum of Even integers is : "+evenSum);
        System.out.println("The Sum of Odd integers is : "+oddSum);
    }
}
